hadoop-hdds/common/src/test/java/org/apache/hadoop/hdds/security/token/TestOzoneBlockTokenIdentifier.java

```diff
@@ -260,15 +259,15 @@ public void testAsymmetricTokenPerf() throws NoSuchAlgorithmException,
   }
 
   @Test
-  public void testSymmetricTokenPerf() {
+  void testSymmetricTokenPerf() {
     String hmacSHA1 = "HmacSHA1";
     String hmacSHA256 = "HmacSHA256";
 
     testSymmetricTokenPerfHelper(hmacSHA1, 64);
     testSymmetricTokenPerfHelper(hmacSHA256, 1024);
   }
 
-  public void testSymmetricTokenPerfHelper(String hmacAlgorithm, int keyLen) {
+  private void testSymmetricTokenPerfHelper(String hmacAlgorithm, int keyLen) {
```

Comment:
@adoroszlai , thanks for fix the issue.  
Does the scope change from public to private solve the problem?

Reply:
No, the problem is fixed by actually using the "useless object" here:

```
    LOG.info("Average token sign time with {}({} symmetric key) is {} ns",
        hmacAlgorithm, keyLen, duration / tokenPasswordSym.size());
```